src/vs/platform/terminal/common/terminal.ts

```diff
@@ -409,6 +409,11 @@ export interface ICreateTerminalOptions {
 	 * The terminal's resource, passed when the terminal has moved windows.
 	 */
 	resource?: URI;
+
+	/**
+	 * Whether or not this terminal should be a split
+	 */
+	isSplitTerminal?: boolean;
```

Comment:
I think this is essentially the same as sideGroup, created https://github.com/microsoft/vscode/issues/129267

Reply:
ok will work on #129267